### ISSUE TITLE
2 quick questions regarding management of comments submitted

### ISSUE THREAD

**Issue by arkid** [stcr/subscribe-to-comments-reloaded#455]
On my site it seems that when users sign up to subscribe to alerts at the point of authoring a comment they don't get an email to confirm a link they can then use at a later point to manage that subscription.

To clarify, all comments need manual approval on this particular WP site, as defined in the discussions admin page setting.

So to clarify, this email exists as part of your plugin or am I imagining functionality that doesn't exist ?

Also, is there a page that users can access and then submit their email address in order to initiate a process of retrieving any subscriptions they may have without the need for an original email that may link them to their unique management page ?

Thanks for your time

**Comment by Reedyseth** [comment]
Hi @arkid 
1. When the user subscribe they don't get an email on the default settings. You have to enable the double check option to get an email for every user that is subscribe.

![image](https://user-images.githubusercontent.com/1228711/49901445-74643b80-fe1e-11e8-8075-960d6dad5fc4.png)

2. When you are subscribe to a given post then the link to manage your subscription will be available on the comment section.

![image](https://user-images.githubusercontent.com/1228711/49901584-cc02a700-fe1e-11e8-8b22-89173835d0e0.png)

**Comment by Reedyseth** [comment]
> To clarify, do all the email alerts have a link to the management of subscriptions page?

Yes, all the notifications has a link to manage the subscriptions.

> Also I'd suggest that even without enabling double check that there is an option to email users confirmation of their subscription. It's much more transparent to do that at the very start of the process. can I request that and if that is going to take a while to do is there an easy way I can add that ?

Correct, I will add the function to be notify when they subscribe even without the double opt-in enable. This does not look complex, idk how I miss that.

Regards.!!

**Comment by arkid** [comment]
What would also be really useful is if the page to manage subscriptions can be accessed without a secure link with unique id.   Ie, you go to the page and if no url params to clarify who you are get parsed, then the page loads and requests your email address.   It then sends a new subscribe link to that address if they are on file.  So this would be a secure way to get people back into the management page if they have lost or can't find their old emails that confirm the management url.      

1) thats not already possible is it ?
2) sound like a good idea? ;)
